lib/matplotlib/backends/backend_wx.py

```diff
@@ -1146,13 +1153,49 @@ def _init_toolbar(self):
 
         self.Realize()
 
+    def _make_toggled_bitmap(self, bitmap):
+        bmp = bitmap.ConvertToImage().ConvertToBitmap()
+        bmp2 = bitmap.ConvertToImage().ConvertToBitmap()
+        brush = wx.Brush(wx.Colour(167*0.85, 167*0.85, 167*0.85))
+        pen = wx.Pen(wx.Colour(167*0.85, 167*0.85, 167*0.85))
+        dc = wx.MemoryDC()
+        dc.SelectObject(bmp)
+        gc = wx.GraphicsContext.Create(dc)
+        gc.SetBrush(brush)
+        gc.SetPen(pen)
+        gc.DrawRoundedRectangle(0, 0, 24, 24, 4)
+        gc.DrawBitmap(bmp2, 0, 0, 24, 24)
+        dc.SelectObject(wx.NullBitmap)
+
+        return bmp
+
     def zoom(self, *args):
+        evt = args[0]
         self.ToggleTool(self.wx_ids['Pan'], False)
         NavigationToolbar2.zoom(self, *args)
+        if 'wxMac' in wx.PlatformInfo:
+            self.SetToolNormalBitmap(self.wx_ids["Pan"],
+                self.bitmaps["Pan"]["Normal"])
+            if evt.IsChecked():
+                self.SetToolNormalBitmap(self.wx_ids["Zoom"],
+                    self.bitmaps["Zoom"]["Toggled"])
+            else:
+                self.SetToolNormalBitmap(self.wx_ids["Zoom"],
+                    self.bitmaps["Zoom"]["Normal"])
```

Comment:
I'd extract this into

~~~
    def _fake_toggle_group(self, elements, active):
        for name in elements:
            self.SetToolNormalBitmap(
                self.wx_ids[name],
                self._bitmaps[name]["Toggled" if name == active else "Normal"]
            )
~~~

so that you can do
~~~
self._fake_toggle_group(["Zoom", "Pan"], active="Zoom")
~~~
here.

Reply:
I've made this change in the latest commit.